src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -450,10 +448,9 @@ def update_app_settings(cmd, resource_group_name, name, settings=None, slot=None
                 slot_cfg_names.app_setting_names.append(slot_setting_name)
             elif not value and slot_setting_name in slot_cfg_names.app_setting_names:
                 slot_cfg_names.app_setting_names.remove(slot_setting_name)
-        app_settings_slot_cfg_names = slot_cfg_names.app_setting_names
         client.web_apps.update_slot_configuration_names(resource_group_name, name, slot_cfg_names)
 
-    return _build_app_settings_output(result.properties, app_settings_slot_cfg_names)
+    return f'Successfully updated settings {list(result.keys())}. Use config appsettings list command to view settings.'
```

Comment:
An Azure CLI command's output should be a value JSON object, so that it can be handled by `--output`.

If you want to convey `Successfully updated settings`, it's better to do it with `logger.warning`.

Reply:
One of our concerns was feedback that we've gotten where users who don't get anything back on the command line are confused about whether their operation has succeeded or not. I've added in the warning for now and omitted the return!

Reply:
Not getting anything is a success. You get nothing when running `rm` or `ln` command, right? 🙂

Reply:
Resolving to continue discussion elsewhere